unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/JoinChannelHUD/JoinChannelComponentController.cs

```diff
@@ -73,7 +74,7 @@ private void OnConfirmJoin(string channelName)
 
         var alreadyJoinedChannel = chatController.GetAllocatedChannelByName(channelName);
 
-        if (alreadyJoinedChannel == null)
+        if (alreadyJoinedChannel == null || (alreadyJoinedChannel != null && !alreadyJoinedChannel.Joined))
```

Comment:
`alreadyJoinedChannel != null` is redundant because it will come to the right side of `||` only when the left-side is false. So you can safely remove it

Reply:
You're right!